ChangeLog

```diff
@@ -1,6 +1,9 @@
 CHANGES
 =======
 
+ADD: Add stats endpoint for uwsgi
+ADD: Print Uwsgi configuration options at docker startup
```

Comment:
No associated issues?

Reply:
Done in https://github.com/telefonicaid/orchestrator/pull/375/commits/4c2f88ace5d36b1649a18b9eb8c813e5a25b4aec